src/apiCalls/loginUser.js

```diff
@@ -1,32 +1,32 @@
 import { signInAction } from "../redux/actions/loginActions";
+import axios from "axios";
 
 export function loginUser(login, pass, setError, navigate) {
-  const options = {
-    method: "POST",
-    headers: { "Content-Type": "application/json" },
-    body: JSON.stringify({ login: login, pass: pass }),
-  };
   return async (dispatch) => {
-    const response = await fetch("http://localhost:3001/login", options);
+    try {
+      const response = await axios({
+        method: "post",
+        url: "http://localhost:3001/login",
```

Comment:
looks like `(http://localhost:3001)` can be extracted into env variable or constant
```suggestion
        url: `${BASE_URL}/login`,
```